### DISCUSSION TITLE
Missing SideNav NPM Package

### DISCUSSION THREAD

**Discussion by acroutworst** [adobe/react-spectrum#2753] [Q&A] [answered]
Hi there 👋

I noticed the SideNav package within the @react-spectrum storybook project. Yet, I'm unable to find the package within [react-spectrum NPM](https://www.npmjs.com/org/react-spectrum). It looks like this package has been in development and closed for a while (refer to #1095).

What is the best way to access this package and is there a way to import from NPM/Yarn?

**Comment by LFDanLu** [comment, marked as answer]
At the moment, the SideNav component is not yet ready for use. The storybook stories were from some initial work done a year or so ago but work on the component has been deprioritized for now. You can track our current priorities via https://github.com/adobe/react-spectrum/projects/1.

**Reply by acroutworst** [reply]
Hi @LFDanLu, thanks for the clarification.

**Reply by conao3** [reply]
@LFDanLu 
Thanks for your work on this project!
The project page (https://github.com/adobe/react-spectrum/projects/1) is unavailable now. What is the current status?
SideNav is common module and I really want to use it.

**Reply by LFDanLu** [reply]
Ah, sorry about that, we've moved to https://github.com/orgs/adobe/projects/19/views/18 as our new project page. SideNav is still deprioritized for the near future unfortunately.

**Reply by conao3** [reply]
OK, thanks!

**Comment by cd-a** [comment]
Looks like this is a far way off from looking at the priorities.

Would be great to have this as it is such a fundamental building block of sites. I wanted to use spectrum but without that component, the whole navigation doesn't work.

I tried using the old package manually but it didn't play well with react router.

**Comment by wingsofovnia** [comment]
While it is not ready, I found it suitable for basic side nav bars. Perhaps, it can be published in nightly versions?

**Reply by LFDanLu** [reply]
Personally, I'm hesitant to expose the SideNav in any fashion on the React Spectrum side of things because we haven't gotten its design/experience vetted by our internal design team. I'd be more willing to expose the aria hooks that power the component so that you all could leverage those to build your own SideNav, but ideally we'd go through a cursory accessibility review for it first since it has been quite some time since anyone on the team has worked on it. I'll bring this up to the team and see what opinions they have regarding this.

**Reply by LFDanLu** [reply]
Talked to the team, we'll be heavily refactoring SideNav and its related hooks in the future so we don't want to expose it at the moment.

**Comment by gar1t** [comment]
Any updates on this? If this isn't going to land anytime soon, could you recommend a pattern using existing components? This is akin to a menu bar in functionality and is a fundamental component for any app dev. Having at least some guidance from the design team on to move this forward would be super helpful!
